### ISSUE TITLE
False positive for multiline imports

### ISSUE THREAD

**Issue by pbludov** [checkstyle/checkstyle#5881]
/var/tmp $ javac TestClass.java

/var/tmp $ cat TestConfig.xml
```
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name = "Checker">
    <module name="TreeWalker">
         <module name="ImportOrder"/>
    </module>
</module>
```

/var/tmp $ cat TestClass.java
```
import                           // OK
java.awt.Button;
import java.awt.Dialog           // OK
    ;
import                           // violation
java.awt.Frame;
import java                      // OK
    .awt.event.
    ActionEvent
    ;
import java.io.File;             // OK
import                           // violation
  static java.awt.Button.ABORT;
import static                    // violation
    java.io.File.createTempFile;
import static javax.swing        // OK
    .WindowConstants.*;

public class TestClass {
}
```

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.10-all.jar -c TestConfig.xml TestClass.java
```
Starting audit...
[ERROR] TestClass.java:6: Extra separation in import group before 'java.awt.Frame' [ImportOrder]
[ERROR] TestClass.java:13: Extra separation in import group before 'java.awt.Button.ABORT' [ImportOrder]
[ERROR] TestClass.java:15: Extra separation in import group before 'java.io.File.createTempFile' [ImportOrder]
Audit done.
Checkstyle ends with 3 errors.
```

There should be no violations. 
At least, there must be some logic. Now it looks like violations happen randomly.

**Comment by romani** [comment]
fix is merged.
